core/lib/zksync_core/src/state_keeper/seal_criteria/conditional_sealer.rs

```diff
@@ -128,11 +128,14 @@ impl SequencerSealer {
         Self { config, sealers }
     }
 
-    fn default_sealers() -> Vec<Box<dyn SealCriterion>> {
+    fn default_sealers(config: &StateKeeperConfig) -> Vec<Box<dyn SealCriterion>> {
         vec![
             Box::new(criteria::SlotsCriterion),
             Box::new(criteria::GasCriterion),
-            Box::new(criteria::PubDataBytesCriterion),
+            Box::new(criteria::PubDataBytesCriterion {
+                max_pubdata_per_da_slot: config.max_pubdata_per_batch,
+                num_da_slots: config.max_number_da_slots,
```

Comment:
1. `max_pubdata_per_da_slot: config.max_pubdata_per_batch` this looks weird, `max_pubdata_per_batch` is not `max_pubdata_per_da_slot`.
2. I think having `max_pubdata_per_da_slot` and `num_da_slots` is useless here, you can only have `max_pubdata_per_batch` stored in config.

Reply:
yea i agree, it seemed like a good idea earlier but state keeper knowing about DA seems like a bad idea